pkg/controller/infinispan/infinispan_controller_test.go

```diff
@@ -61,6 +61,10 @@ func (m mockCluster) GetClusterMembers(podName string) ([]string, error) {
 	return nil, errors.New("error in getting view")
```

Comment:
How about here? Could this be when a network condition or some other fault interrupts the operator ability to get the cluster view?

Reply:
@oraNod it's a mock test, please ignore.